### PR TITLE
Adding AOF section to `Managing Dragonfly`

### PR DESCRIPTION
**Pull request by joswayski**
Based on this comment here:
https://discord.com/channels/981533931486724126/981533931960676364/1118615556371783720

CleanShot 2024-03-23 at 18 53 08@2x

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13032 @@
+{
+  "name": "docs",
```
Comment (romange) [inline package-lock.json:2]: is it necessary to add to the PR?

File: yarn.lock [hunk 2]
```diff
@@ -4,138 +4,186 @@
 
 "@algolia/autocomplete-core@1.9.3":
   version "1.9.3"
+  resolved "https://registry.npmjs.org/@algolia/autocomplete-core/-/autocomplete-core-1.9.3.tgz"
```
Comment (romange) [inline yarn.lock:7]: is it necessary to add to the PR?

File: docs/managing-dragonfly/append-only.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 5
+---
+
+# Append Only Mode (AOF)
+
+At this time, Dragonfly does not support Append Only Mode (AOF). The primary reason is that the current implementation of the RDB snapshotting mechanism is not compatible with the AOF mode. The AOF mode requires a different approach to snapshotting and recovery, which is not implemented in Dragonfly, however, it is something we are considering for future versions. Please join the discussion on the [Dragonfly Discord](https://discord.com/invite/HsPjXGVH85) if you have any thoughts or suggestions on this topic.
```
Comment (romange) [inline docs/managing-dragonfly/append-only.md:7]:
```suggestion
At this time, Dragonfly does not support Append Only Mode (AOF). The primary reason is that it has not been a high priority request in the community. The AOF mode in Dragonfly will require a different approach than in Redis and is more complicated to implement, however, it is something we are considering for future versions. Please join the discussion on the [Dragonfly Discord](https://discord.com/invite/HsPjXGVH85) if you have any thoughts or suggestions on this topic.
```

**Review by romange — COMMENTED** [review]
Thank you for adding this

File: package-lock.json (hunk 1 above)
Comment (joswayski) [reply package-lock.json:2]: No - i can remove - was a consequence of running `npm install` :D

**Comment by Niennienzz** [comment]
Thanks for the contribution. Please, I will take a look at the details before merging.

**Comment by romange** [comment]
LGTM from my side.

**Review by romange — APPROVED** [review]

**Comment by romange** [comment]
Thank you!

**Pull request merged**